src/daft-sql/src/catalog.rs

```diff
@@ -4,11 +4,11 @@ use daft_logical_plan::{LogicalPlan, LogicalPlanRef};
 
 /// A simple map of table names to logical plans
 #[derive(Debug, Clone)]
-pub struct SQLCatalog {
+pub struct InMemoryCatalog {
     tables: HashMap<String, Arc<LogicalPlan>>,
 }
```

Comment:
I see this holds `Arc<LogicalPlan>` whereas _DaftMetaCatalog_ holds just `LogicalPlanBuilder`. It looks like you've been able to replace _DaftMetaCatalog_ in the planner as well as the python API. If possible could you please remove SQLCatalog/InMemoryCatalog in this PR? 

If a `String, Arc<LogicalPlan>` map is needed somewhere, then I suggest it be internalized to the requiring module as well as dropping the "catalog" name to ease confusion.

Reply:
oh whoops, bad merge. I meant to delete this file